config/initializers/assets.rb

```diff
@@ -32,5 +32,5 @@
 Rails.application.config.assets.paths += %W(
   #{__dir__}/../../node_modules/govuk-frontend/assets/images
   #{__dir__}/../../node_modules/govuk-frontend/assets/fonts
```

Comment:
Do we know if we use the above two includes at all? I wonder if it'd cause pain to change them so those files are included via `govuk-frontend/assets/images/file`

Reply:
> Only thing I'd suggest is updating the changelog and noting this is a breaking change for anyone including govuk-frontend through this gem

Updated the changelog, but I don't see why it would be a breaking change. It still consumes `govuk-frontend` via `govuk_publishing_components` gem it's just that now is doing it without using relative paths.